README.md

```diff
@@ -38,26 +38,21 @@ Read input files as binary and pass it to the `DocxMerger` constructor fuction a
 Then call the save function with first argument as `nodebuffer`, check the example below.
 
   ```javascript
-  const DocxMerger = require('docx-merger');
-
-  const fs = require('fs');
-  const path = require('path');
-
-  const file1 = fs
-      .readFileSync(path.resolve(__dirname, 'template.docx'), 'binary');
-
-  const file2 = fs
-      .readFileSync(path.resolve(__dirname, 'template1.docx'), 'binary');
-
-  const docx = new DocxMerger({},[file1,file2]);
-
-
-  //SAVING THE DOCX FILE
-
-  docx.save('nodebuffer',function (data) {
-      // fs.writeFile("output.zip", data, function(err){/*...*/});
-      fs.writeFile("output.docx", data, function(err){/*...*/});
-  });
+  const DocxMerger = require('./../src/index');
+
+const fs = require('fs');
+const path = require('path');
+
+(async () => {
+    const file1 = fs.readFileSync(path.resolve(__dirname, 'template.docx'), 'binary');
```

Comment:
Do we want to hardcode filenames?

Reply:
This is an example of the README, and the filenames were hardcoded so I think it's OK!